test/integration/auth.spec.ts

```diff
@@ -1672,6 +1742,34 @@ describe('admin.auth', () => {
         });
     });
 
+    it('updateTenant() should not update tenant reCAPTCHA config is undefined', () => {
+      expectedUpdatedTenant.tenantId = createdTenantId;
+      const updatedOptions2: UpdateTenantRequest = {
+        emailSignInConfig: {
+          enabled: true,
+          passwordRequired: false,
+        },
+        multiFactorConfig: deepCopy(expectedUpdatedTenant2.multiFactorConfig),
+        // Test clearing of phone numbers.
+        testPhoneNumbers: null,
```

Comment:
I feel like this is testing multiple things. Would it make more sense to just limit this test case only for `recaptchaConfig: undefined` ?

Reply:
I kept only recaptchaConfig and an extra displayName in the new version so that the request data sent is not empty. Let me know your thought on this.